File: my-folder/1078-remove-outermost-parentheses/solution.cpp
class Solution {
public:
    string removeOuterParentheses(string s) {
        string ans;
        int count = 0;
        for(char c:s){
            if(c=='(' && count++>0) ans+=c;
            if(c==')' && count-->1) ans+=c; 
        } 
        return ans;
        
    }
};

File: my-folder/1078-remove-outermost-parentheses/solution.java
class Solution {
    public String removeOuterParentheses(String s) {
        StringBuilder ans = new StringBuilder();
        int count = 0;
        for(char c:s.toCharArray()){
            if(c=='(' && count++>0) ans.append(c);
            if(c==')' && count-->1) ans.append(c);
        }
        return ans.toString();
        
    }
}
